osconfig_tests/test_suites/package_management/package_management_test_data.go

```diff
@@ -25,13 +25,18 @@ import (
 	api "google.golang.org/api/compute/v1"
 )
 
+type platformPkgManagerTuple struct {
+	platform   string
+	pkgManager string
```

Comment:
I don't see pkgManager used anywhere, is there a reason we need this?

Reply:
it is used to fetch the startup script, since the startup scripts are dependent on package manager
https://github.com/GoogleCloudPlatform/compute-image-tools/pull/682/files#diff-1f8f3e56a68ead097f839c047dbc57b2R113